tests/behaviour/steps/cubes.py

```diff
@@ -103,10 +65,12 @@ def step_impl(context):
 
     for template_file in template_files:
         relative_file_path = str(template_file.relative_to(templates_dir))
-        paths_to_mock["https://raw.githubusercontent.com/GSS-Cogs/csvcubed/main/src/csvcubed/readers/cubeconfig/v1_0/templates/"+ relative_file_path] = template_file
+        PATHS_TO_MOCK[
```

Comment:
I don't think we should be updating what is essentially constant state here, if we're going to continue doing this can you create a deep copy of `PATHS_TO_MOCK` before updating it?